production/scrapers/south_carolina.R

```diff
@@ -85,7 +103,7 @@ south_carolina_extract <- function(x){
         select(-ends_with("Drop"))
 }
 
-#' Scraper class for general south_carolina COVID data
+#' Scraper class for general south_NOTE: This information is current as of 9:46 a.m. on July 06, 2021 carolina COVID data
```

Comment:
seems to be an oops copy-paste that happened here?

Reply:
Thanks changed!